packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,33 +1,70 @@
 // ----------------
-// Variables
+// 1.  CSS Variables
+// 2.  Dark Mode
+// 3.  Autocomplete
+// 4.  Panel
+// 5.  Sources
+// 6.  Hit Layout
+// 7.  Panel Header
+// 8.  Panel Footer
+// 9.  Detached Mode
+// 10. Gradients
+// ----------------
+
+// Note:
+// This theme is trying to reflect the markup of autocomplete with SCSS indentation.
+// We use the SASS `@at-root` function to remove unecessary specificities.
+
+// ----------------
+// 1. CSS Variables
 // ----------------
 :root {
+  // input
+  --aa-search-input-height: 44px;
```

Comment:
I also tend to use `rem` so I agree. I believe @Shipow focused more on the design aspect and initially chose `px`.

@Shipow Are you OK with moving to `rem`s?

Reply:
I have a really bad experience with rem, happy to give you a chance to convince me, I might have miss something.

Reply:
@Shipow Was it with `em` or `rem`? `em` can be confusing but `rem` is only a multiple of the root font-size, so it's straightforward.

Reply:
I guess I just prefer to setup the size of the search experience independently than the rest of the ui. Maybe most of the time it will be the correct setup but if you need to change it, it becomes a pain. Like the ais screenshot i've sent you in dm that kept rem for the highlighting even if you set the parent to the right font size.

Reply:
em would solve that, it's always relative to the parent

Reply:
I guess we need to experiment some more to make a decision.